src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetViewModels.cs

```diff
@@ -111,3 +111,106 @@ public DataSetPaginatedListViewModel(List<DataSetViewModel> results, PagingViewM
     {
     }
 }
+
+public enum DataSetVersionType
+{
+    Major,
+    Minor
+}
```

Comment:
Similar to other comments, would move this into `Public.Data.Model` and make this part of the data model. It's feasible that this would be useful in other places so it makes sense to go there.